src/ml/neural_net/mps_layer_conv_padding.h

```diff
@@ -0,0 +1,30 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+API_AVAILABLE(macos(10.14))
+@interface ConvolutionPadding : NSObject <MPSNNPadding>
```

Comment:
Objective C doesn't have namespaces, so it's incumbent on library code to use prefixes to reduce the likelihood of namespace collisions. I suggest `TCMPS`, so we can rename this class `TCMPSConvolutionPadding`

Reply:
Got it. Updated the library name to avoid confusion.